src/utils/useWeb3Connect.js

```diff
@@ -387,10 +424,10 @@ function useWeb3Connect() {
       );
       // console.log({ iteration, address });
       let tempCurrentVote = 0;
-      if (connected && address) {
+      if (connected && addr) {
```

Comment:
Any reason to change this?

Reply:
The reason is that the fetch proposals is now also called from the initialization function. It was relying on the state variable daoContractReadOnly that is set on that function. The setDaoContractReadOnly is not updating the state on the same loop, so the value on this function was undefined. Therefore we send the contract from that init function until this call. 

I know that this is not beautiful, but I struggled with this file and flow...